src/main/scala/zio/prelude/CommutativeEither.scala

```diff
@@ -1,14 +1,13 @@
 package zio.prelude
 
-import scala.concurrent.{ ExecutionContext, Future, Promise }
-
 import zio.ZIO
 import zio.prelude.coherent.CommutativeEitherDeriveEqualInvariant
 import zio.stream.{ ZSink, ZStream }
 import zio.test.TestResult
 import zio.test.laws._
 
 import scala.annotation.implicitNotFound
+import scala.concurrent.{ ExecutionContext, Future, Promise }
```

Comment:
Would suggest we follow zio style of `java` first, then `scala`, then `zio` and others.

Reply:
We can definitely do that. But please note, that this order is different from IntelliJ IDEA's default. That means they will fight each other when ordering imports. And IDEA is the most prevalent IDE, so it may be confusing/difficult for some (many?) people.

Reply:
That's the order we have used for ZIO and I think it has worked okay. One think that helps was that we added a command alias `prepare` that ran both Scalafmt and Scalfix so we could tell contributors to do this one thing before pushing to make sure that their code worked.